src/server/handlers/github.js

```diff
@@ -137,3 +144,35 @@ const getRequest = (owner, name, token, secret) => {
     }
   };
 };
+
+/*
+ * parse_link_header()
+ *
+ * Parse the Github Link HTTP header used for pageination
+ * http://developer.github.com/v3/#pagination
+ *
+ * Modified by kfenn to return page numbers instead of urls
+ */
+const parseLinkHeader = (header) => {
+  if (header.length == 0) {
+    throw new Error('input must not be of zero length');
+  }
+
+  // Split parts by comma
+  let parts = header.split(',');
+  let links = {};
+  // Parse each part into a named link
+  for (let index=0; index<parts.length; index++) {
+    let section = parts[index].split(';');
+    if (section.length != 2) {
+      throw new Error('section could not be split on ";"');
+    }
+    let number = parseInt(section[0]
+                  .replace(/per_page=(\d+)/, '') // Make sure per_page does not match
+                  .match(/page=(\d+)/)[1]);
+    let name = section[1].replace(/rel="(.*)"/, '$1').trim();
+    links[name] = number;
+  }
+
+  return links;
+};
```

Comment:
@cjwatson: True. For something this small, I'd prefer to copy an example from the community and add it to our own source code rather than installing a module. It's simple enough that we can maintain it ourselves while avoiding the risk of something happening upstream, `left-pad` style. I will probably replace this with a helper we can re-use in `src/server/helpers` in time.